docs/source/fx.rst

```diff
@@ -298,6 +290,245 @@ Miscellanea
    -  Annotations on local names within a function are not currently
       supported.
 
+Writing Transformations
+-----------------------
+
+TODO
+
+
+Debugging
+-----------
+
+Introduction
+^^^^^^^^^^^^^^^^
+
+Once we've generated Python code from our symbolically-traced Module,
+we'll want to verify that the proper semantics were preserved. If they
+weren't, we may need to do some debugging. The key is to work
+backwards: first, check the results of the generated module, then debug
+the generated code, then debug the process of transformations that lead
+to the generated code.
+
+If you’re not familiar with debuggers, please see the auxiliary section
+:ref:`Available Debuggers`.
+
+Debugging the Generated Code
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Because FX generates the ``forward()`` function on GraphModules, using
+traditional debugging techniques like ``print`` statements or ``pdb`` is
+not as straightfoward. Luckily, we have several techniques we can use
+for debugging the generated code.
+
+Use ``pdb``
+~~~~~~~~~~~~~
+Invoke ``pdb`` to step into the running program. Although the code that
+represents the FX graph is not in any source file, we can still step
+into it manually using ``pdb`` when the forward pass is invoked.
+
+::
+
+    x, y = torch.rand(2, 3), torch.rand(2, 3)
+    m = symbolic_trace(M())
+    import pdb; pdb.set_trace()
```

Comment:
Might want to add a comment saying something like "When this line is executed at runtime, we will be dropped into an interactive `pdb` prompt. We can use the `step` or `s` command to step into the execution of the next line"